shotgun_api3/shotgun.py

```diff
@@ -593,6 +599,9 @@ def find(self, entity_type, filters, fields=None, order=None,
         :param include_archived_projects: Optional, flag to include entities
         whose projects have been archived
 
+        :param additional_filter_presets: Optional, list of filter presets to
+        apply.
+
```

Comment:
Also, might be useful to add a small note here that presets get ANDed together with the rest of the filters?